### ISSUE TITLE
Cloud Profile 2.18.1: Consider removing

### ISSUE THREAD

**Issue by alex941115** [appdefensealliance/ASA-WG#109]
[2.18.1 - Ensure IAM Users Receive Permissions Only Through Groups](https://github.com/appdefensealliance/ASA-WG/blob/main/Cloud%20App%20and%20Config%20Profile/Cloud%20App%20and%20Config%20Test%20Guide.md#2181-ensure-iam-users-receive-permissions-only-through-groups)

Consider removing. I think the intent behind it is sound but it’s pretty common to just use managed policies as well, and if you have a small account without many users then I don’t think it adds much.

**Comment by rdegraaf** [comment]
Agreed.  The reasoning for using Groups is that it makes permissions easier to manage if you have a lot of Users with the same permissions.  This became a lot less important when Managed permissions policies were introduced, since now you can control the permission policy in one place regardless of how many Users attach it.  But there is still value in Groups for cases where you have a bunch of Users all with the same *set* of permission policies.

However, if you have a one-off User, then there's no value in creating a one-off Group for that User's permissions.  For that matter, there's also no value in creating a one-off Managed permission policy either; just use an in-line permission policy.

I'm not sure why so many security guides still have this requirement and the similar "don't use in-line permission policies" requirement.  IHMO, both should be replaced with something along the lines of "don't create duplicate permission policies".  Of course, that's a little harder to check; you need to pull every permission policy and compare them, rather than just doing a simple check on every User.

**Comment by 8radree** [comment]
We will review past issues when we update to the latest version of the CIS Controls.
